styles/rentry.co/catppuccin.user.css

```diff
@@ -0,0 +1,568 @@
+    }
+    .btn {
+      background: @base !important;
+      color: @text !important;
+      &:hover {
+        background: @surface0 !important;
+        color: @text !important;
+      }
+    }
+    .btn-success {
+      &,
+      &:hover {
+        color: @green !important;
+      }
+      &#submitButton {
+        background: @base !important;
+        &:hover {
+          background: @surface0 !important;
+        }
+      }
+    }
+    .btn-danger {
+      &,
+      &:hover {
+        color: @red !important;
+      }
+    }
+    .clipboard {
+      color: @text;
+      background: none;
+    }
+    #progressBar {
+      background-color: @blue !important;
+    }
+    .full-contrast:not(.active) path {
+      fill: @overlay2 !important;
+    }
+    .full-contrast.active path {
+      fill: @peach !important;
+    }
+    .edit-code {
+      background: @mantle;
+      color: @text !important;
+    }
+    .cb-tooltiptext {
+      color: @text !important;
+      background: @surface0 !important;
+    }
+    .cb-tooltiptext-bottom::after {
+      border-color: transparent transparent @surface0;
+    }
+    .headerlink {
+      color: @overlay1 !important;
+    }
+    #submitButton {
+      background: @mantle !important;
+    }
+    // claim-guide page
+    .color-change {
+      &[style="color:grey"] {
+        color: @overlay2 !important;
+      }
+      &[style="color:#039205"] {
+        color: @green !important;
+      }
+    }
+    // delete modal
+    .modal-content {
+      background: @crust;
+      color: @text;
+    }
+    .modal-body {
+      color: @overlay2;
+    }
+    .modal-header {
+      color: @text;
+    }
+    .modal-header,
+    .modal-footer {
+      border-color: @surface1 !important;
+    }
+    #deleteButton {
+      color: @red !important;
+    }
+    // footer
+    a {
+      color: @blue;
+
+      &:hover {
+        color: @sky !important;
+      }
+    }
+    .text-muted {
+      color: @overlay2 !important;
+    }
+    .text-primary {
+      color: @blue !important;
+    }
+    .text-success {
+      color: @green !important;
+    }
+    .text-warning {
+      color: @yellow !important;
+    }
+    .text-danger {
+      color: @red !important;
+    }
+    // editor
+    .CodeMirror-line::selection,
+    .CodeMirror-line > span::selection,
+    .CodeMirror-line > span > span::selection {
+      background-color: fade(@overlay2, 25%);
+    }
+    .cm-formatting-admonition {
+      color: @yellow !important;
+    }
+    .cm-mark {
+      color: @yellow !important;
+    }
+    .cm-link {
+      color: @blue !important;
+    }
+    .cm-url {
+      color: @sky !important;
+    }
+    .cm-formatting-toc {
+      color: @mauve !important;
+    }
+    .cm-variable-2 {
+      color: @green !important;
+    }
+    .cm-keyword {
+      color: @pink !important;
+    }
+    .cm-meta {
+      color: @subtext0 !important;
+    }
+    .cm-comment {
+      color: @pink !important;
+    }
```

Comment:
Comments are pink? 🤔

Reply:
code comments are not pink. despite the confusing name, this class is for inline code and code blocks in the markdown editor
![image](https://github.com/user-attachments/assets/431dbd07-5712-4228-83a3-6369e48b14fe)
![image](https://github.com/user-attachments/assets/60228b3d-ca57-466d-aede-838ce2e6458b)
![image](https://github.com/user-attachments/assets/d77901a2-82fa-40ed-b1e3-70126e0f2006)
code comments are here: https://github.com/catppuccin/userstyles/pull/1227/files#diff-fb69bb938dda439c6f2e4b2c30f44effaf6bbf727d788844a42bbf6b4c49c5e0R407-R414 and are colored in `@overlay2`